docs/sdk/php.md

```diff
@@ -0,0 +1,90 @@
+## PHP SDK
+
+### Installation
+
+Installation is best done using [composer](https://getcomposer.org/)
+
+Create a composer.json file with the following:
+
+```json
+{
+    "repositories": {
+        "tulip/hydra-php-sdk": { "type": "vcs", "url": "git@git.internal.tulip.io:shared/composer-libraries/hydra-php-sdk.git" }
+    },
+    "require": {
+        "tulip/hydra-php-sdk": "dev-master"
+    }
+}
+```
+
+Then run `composer install` and include the generated autoload file in your script.
+
+### Configuration
+
+#### OAuth2 configuration
+
+We need OAuth2 capabilities in order to make authorized API calls. You can either write your own OAuth2 mechanism or
+use an existing one that has been preconfigured for use with Hydra. Here we use a modified version of the league OAuth2
+client that has had this work done for us.
+
+```sh
+composer require tulip/oauth2-hydra
```

Comment:
Same thing here I guess?

Reply:
Different repo. I can change the org to ory but maybe it's better if you take ownership if it's official. You can fork this guy https://github.com/tulipretail/oauth2-hydra, change tulip/oauth2-hydra to ory/oauth2-hydra in `composer.json` and submit it to packagist.

You could also make it official by making a PR to add it to the league's list https://github.com/thephpleague/oauth2-client/blob/master/docs/providers/thirdparty.md